README.md

```diff
@@ -202,6 +202,11 @@ participated in that process.
 
 [Query issues for "Meeting"][]
 
+## Thanks to all the contributors ❤
+<a href = "https://github.com/nodejs/nodejs.dev/graphs/contributors">
+   <img src = "https://contrib.rocks/image?repo=nodejs/nodejs.dev"/>
```

Comment:
```suggestion
  <img src = "https://contrib.rocks/image?repo=nodejs/nodejs.dev"/>
```

Reply:
Done @ljharb . Please re-review this PR.